scripts/trigger-dependent-ci-builds.sh

```diff
@@ -3,21 +3,49 @@
 BUILD_MESSAGE="Test ${TRAVIS_REPO_SLUG} ${TRAVIS_COMMIT_RANGE}"
 
 # Trigger HELICS-FMI build
-body='{
-"request": {
-"message":"'
-body+="${BUILD_MESSAGE}"
-body+='",
-"branch":"master"
-}}'
+#body='{
```

Comment:
are we not triggering the FMI build any more?

Reply:
It looks like at some point the API token stopped working. I thought I'd seen something about Travis API tokens expiring every few months, but it may also be GH permissions for the HELICS-bot account. I'm thinking of switching HELICS-FMI builds over to Azure so that most of the setup for the dependent repositories is the same -- and the Azure API rate limit for queuing builds is more generous (Travis API only allows 3 queued builds/hour).

Reply:
that sounds reasonable,  we probably should still have some of the builds of HELICS-FMI on travis though.  But the triggered builds seems like a good idea to make consistent across all the repos.